app/src/main/res/layout/dialog_play_offline.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:padding="20dp">
+
+
+    <!-- todo: video info could be added
```

Comment:
Seems like a good idea to display information about the downloaded video here, title, caption language (if any) and resolution / audio quality should be sufficient.